Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/BaseController.java

```diff
@@ -143,6 +143,7 @@ protected String getURL(final String nativeType) {
       case EPML_2_0:
         url = "http://b3mn.org/stencilset/epc#";
         break;
```

Comment:
if you have made changes here. It could be better to use url variable and return it at the end.
Rather thsan default everywhere